docs/source/guide/events.rst

```diff
@@ -212,11 +212,40 @@ Boto3 specific events
+| ``provide-client-params``   | 2 *   | `Location <https://github.com/boto/botocore/blob/aa63a377c6c9a78c87f55e5c0fc6ca6643198f3b/botocore/client.py#LL1018C1-L1023C10>`_    |
++-----------------------------+-------+--------------------------------------------------------------------------------------------------------------------------------------+
+| ``before-parameter-build``  | 3     | `Location <https://github.com/boto/botocore/blob/892c5bc73606ec02a6eff2522da1decff88d1c48/botocore/client.py#LL1026C1-L1031C10>`_    |
++-----------------------------+-------+--------------------------------------------------------------------------------------------------------------------------------------+
+| ``before-call``             | 4     | `Location <https://github.com/boto/botocore/blob/5599853483f89d9d687c60ea071c37cd220d5569/botocore/client.py#LL929C1-L937C10>`_      |
++-----------------------------+-------+--------------------------------------------------------------------------------------------------------------------------------------+
+| ``request-created``         | 5     | `Location <https://github.com/boto/botocore/blob/aa63a377c6c9a78c87f55e5c0fc6ca6643198f3b/botocore/endpoint.py#LL131C1-L138C14>`_    |
++-----------------------------+-------+--------------------------------------------------------------------------------------------------------------------------------------+
+| ``before-send``             | 6     | `Location <https://github.com/boto/botocore/blob/4691c4d6ac0cc3461fb5c40dc6908222866a525d/botocore/endpoint.py#LL277C1-L278C78>`_    |
++-----------------------------+-------+--------------------------------------------------------------------------------------------------------------------------------------+
+| ``needs-retry``             | 7     | `Location <https://github.com/boto/botocore/blob/4691c4d6ac0cc3461fb5c40dc6908222866a525d/botocore/endpoint.py#LL353C1-L362C10>`_    |
++-----------------------------+-------+--------------------------------------------------------------------------------------------------------------------------------------+
+| ``after-call``              | 8 *   | `Location <https://github.com/boto/botocore/blob/fc30d05149c248d5e601bea37be422e97c5ad7ee/botocore/client.py#LL947C1-L955C10>`_      |
++-----------------------------+-------+--------------------------------------------------------------------------------------------------------------------------------------+
+| ``after-call-error``        | 8 *   | `Location <https://github.com/boto/botocore/blob/892c5bc73606ec02a6eff2522da1decff88d1c48/botocore/client.py#LL968C1-L975C14>`_      |
++-----------------------------+-------+--------------------------------------------------------------------------------------------------------------------------------------+
+
+.. note::
+
+  2 \* - ``creating-resource-class`` is emitted when using a service client and
+  ``provide-client-params`` is emitted when using a service resource.
```

Comment:
@dlm6693 These events pointed out here share the same order number because they're emitted in specific cased.  I'll try to make that more clear.

@nateprewitt  Thanks for catching! Thats was probably a mistake on my end when reordering the events in the notes section.